components/shell/fish/manifests/sample-manifest.p5m

```diff
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright 2017 <contributor>
+# Copyright 2017 Longrin Wischnewski
```

Comment:
Don't do this. sample-manifest shouldn't be changed. It's just a reference for future use. Only p5m file is used  by build system.